assets/src/apps/authoring/components/RightMenu/RightMenu.tsx

```diff
@@ -62,14 +66,23 @@ const RightMenu: React.FC<any> = () => {
   const screenPropertyChangeHandler = useCallback(
     (properties: any) => {
       if (currentActivity) {
+        const currentSequence = findInSequence(sequence, currentSequenceId);
```

Comment:
this stuff is what I'm talking about moving into a separate thunk, so that below it can be like
```
if (titleChanged) {
  dispatch(updateSequenceItemFromActivity({ activity: cloneActivity}));
}
```
so this method only needs to know that the sequence needs to be updated, not the details of *how*